packages/protocol/src/abstracts/BaseVault.sol

```diff
@@ -39,13 +40,24 @@ abstract contract BaseVault is ERC20, SystemAccessControl, PausableVault, VaultP
   error BaseVault__withdraw_slippageTooHigh();
   error BaseVault__redeem_slippageTooHigh();
 
-  IERC20Metadata internal immutable _asset;
+  /**
+   * @dev `VERSION` of this vault.
+   * Software versioning rules are followed: v-0.0.0
+   *  v-MAJOR.MINOR.PATCH
+   *  MAJOR version when you make incompatible ABI changes
+   *  MINOR version when you add functionality in a backwards compatible manner.
+   *  PATCH version when you make backwards compatible fixes.
+   */
+  string public constant VERSION = "v-0.0.0";
```

Comment:
Why not just "0.0.0"?

It's a bit weird all zeroed; maybe we start with "0.0.1"?

Reply:
updated.